src/makeCalculator.js

```diff
@@ -3,8 +3,51 @@
 /**
  * @return {object}
  */
+
+/* eslint-disable no-console */
+
 function makeCalculator() {
```

Comment:
The function name `makeCalculator` is appropriate as per the naming convention specified in the checklist.